client/components/ui/date-picker.js

```diff
@@ -8,7 +8,10 @@ export default class DatePicker extends React.Component {
     }
 
     componentDidMount() {
-        this.pickadate = $(this.refs.elem.getDOMNode()).pickadate().pickadate('picker');
+        let pickerOptions = this.generateLocalizationObject();
+        pickerOptions = this.setMaxorMin(pickerOptions, this.props);
+        this.pickadate = $(this.refs.elem.getDOMNode()).pickadate(pickerOptions)
```

Comment:
Can you put `this.refs.elem.getDOMNode()` into a local variable, so this statements fits on a single line, please?